frontend/src/api/k8s/utils.ts

```diff
@@ -74,3 +74,24 @@ export const getshmVolume = (sizeLimit?: string): Volume => ({
   name: 'shm',
   emptyDir: { medium: 'Memory', ...(sizeLimit && { sizeLimit }) },
 });
+
+export const parseCommandLine = (input: string): string[] => {
+  const args: string[] = [];
+  const regex = /(?:[^\s"']+|"[^"]*"|'[^']*')+/g;
+  let match: RegExpExecArray | null;
+
+  while ((match = regex.exec(input)) !== null) {
```

Comment:
Looking closer at this - what iterates this loop? does `regex.exec(input)` with the same input multiple times step to the next match? A quick glance at docs says yes - interesting, I haven't used it that way before.

I wonder if it would be simpler to use `regex.match` instead of `regex.exec` to just get back an array of all the matches, and then do a `forEach` or `reduce` on those? I'm nitpicking though, it's probably fine as-is.

Reply:
I'm fine with either solution, that's just the one that it gave me

Reply:
If it works it works, but I may tinker with an alternative when I have more time to review. Did you still plan on adding / have time to add unit tests here?